force-app/main/default/classes/FeatureParameterMapper.cls

```diff
@@ -117,7 +117,10 @@ public virtual with sharing class FeatureParameterMapper {
             Org_CountActiveCourseConnectionRecordTypes,
             SettingsHealthCheckLastRunDate,
             UsingOldContactEthnicity,
-            UsingOldCourseDescription
+            UsingOldCourseDescription,
+            Has_AffiliationsRelatedToDeletedProgramEnrollments,
+            Has_Affiliation_Start_Date,
```

Comment:
It seems the naming convention is not that straightforward. Should we follow `HasDefaultAdministrativeAccountModel` naming? If so, then we would have `HasAffiliationStartDate` feature parameter name. I think this one is preferred.
If we follow `Has_AffiliationsRelatedToDeletedProgramEnrollments`, then we should have `Has_AffiliationStartDate` name. Thoughts?

Reply:
Sure thing, that naming convention also works ([reference commit](https://github.com/SalesforceFoundation/EDA/pull/1923/commits/375428e2e10d4563e25b7ab440946c212accee68)).